doc/tor-genkey.1.txt

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) The Tor Project, Inc.
+// See LICENSE for licensing information
+// This is an asciidoc file used to generate the manpage/html reference.
+// Learn asciidoc on http://www.methods.co.nz/asciidoc/userguide.html
+:man source:   Tor
+:man manual:   Tor Manual
+tor-genkey(1)
+==============
+Suphanat Chunhapanya
+
+NAME
+----
+tor-genkey - utility to generate various keys for tor
+
+SYNOPSIS
+--------
+**tor-genkey** [-t|--type __keytype__] [-o|--out __filename__]
```

Comment:
Here is another thought now. What about a git alike style command?

`tor-genkey hs clientauth descriptor -o <filename>` type of command? (not final format, just to give an idea)